packages/cozy-client/src/models/document/qualification.js

```diff
@@ -36,12 +36,16 @@ export class Qualification {
       /**
        * @type {string?}
        */
-      this.purpose = attributes.purpose || qualification.purpose
-      this.sourceCategory =
+      const purpose = attributes.purpose || qualification.purpose
+      if (purpose) this.purpose = purpose
```

Comment:
That's a nit, but I kind of dislike this syntax, especially with no indent. I find it quite hard to read, even more when there are several following with no line break. Good ol' verbose seems more readable to me:
```js
if (purpose) {
  this.purpose = purpose
}
```